src/wp-includes/blocks.php

```diff
@@ -176,8 +176,15 @@ function register_block_style_handle( $metadata, $field_name ) {
 		array(),
 		filemtime( $style_file )
 	);
+
+	wp_style_add_data( $style_handle, 'path', $style_file );
+
 	if ( file_exists( str_replace( '.css', '-rtl.css', $style_file ) ) ) {
 		wp_style_add_data( $style_handle, 'rtl', 'replace' );
+
+		if ( is_rtl() ) {
+			wp_style_add_data( $style_handle, 'path', str_replace( '.css', '-rtl.css', $style_file ) );
```

Comment:
Have you considered introducing `path-rtl` instead? It's probably fine as is, but I'm not quite sure why `wp_style_add_data` has this `rtl` data in the first place.